modules/data/event-hub/main.bicep

```diff
@@ -0,0 +1,245 @@
+@description('Optional. Name for the Event Hub cluster, Alphanumerics and hyphens characters, Start with letter, End with letter or number.')
+param clusterName string = 'null'
```

Comment:
```suggestion
@description('Optional. Prefix for the Event Hub cluster name')
param prefix string = 'evhc'

@description('Optional. Name for the Event Hub cluster, Alphanumerics and hyphens characters, Start with letter, End with letter or number.')
param name string = '${prefix}${uniqueString(resourceGroup().id, location)}'
```

Reply:
similar recommendation as in the other PR. I like to always use the param 'name' for the primary resource.